nixos/doc/manual/configuration/gpu-accel.xml

```diff
@@ -101,4 +101,89 @@ ROCR_EXT_DIR=`nix-build '&lt;nixpkgs&gt;' --no-out-link -A rocm-runtime-ext`/lib
       </para>
     </section>
   </section>
+
+  <section xml:id="sec-gpu-accel-vulkan">
+    <title>Vulkan</title>
+
+    <para>
+      <link xlink:href="https://en.wikipedia.org/wiki/Vulkan_(API)">Vulkan</link> is a
+      graphics and compute API for GPUs. It is used directly by games or indirectly though
+      compatibility layers like DXVK.
+    </para>
+
+    <para>
+     By default, if <xref linkend="opt-hardware.opengl.driSupport"/> is enabled,
+     <package>mesa</package> is installed and provides Vulkan for supported hardware.
+    </para>
+
+    <para>
+      Similar to OpenCL, Vulkan drivers are loaded through the <emphasis>Installable Client
+      Driver</emphasis> (ICD) mechanism. ICD files for Vulkan are json files that specify
+      the path to the driver library and the supported Vulkan version. All successfully
+      loaded drivers are exposed to the application as different GPUs.
+      In NixOS, there are two ways to make ICD files visible to Vulkan applications.
```

Comment:
You hint that there's two ways, but do a deep dive into the first option immediately. Might be nice to give a small indication as what both options are, then have the more detail explanations after.
```suggestion
      In NixOS, there are two ways to make ICD files visible to Vulkan applications: an 
      environment variable, and a module option. The first is through the ...
```

Reply:
Good idea, I also split it into two paragraphs.